bin/rdygo-service.js

```diff
@@ -24,16 +22,9 @@ if (!process.argv.slice(2).length) {
   // parse json config file
   configuration = myConfig.init({
     path: path.resolve(program.configuration),
-    env: program.environment
+    env: program.environment,
   });
 
-  // start server using current configuration
-  createWorker(configuration, function(errWrker, server) {
-    if (errWrker) throw errWrker;
-    server
-      .listen(configuration.port, configuration.ip, function(err) {
-        if (err) throw err;
-        winston.info('Server listening on ', configuration.port, ' port');
-      });
-  });
+  // execute the cluster
+  cluster(configuration);
```

Comment:
ok I will remove the cluster logic, just a note here IMHO: using Nginx or any reverse proxy server will help us to achieve the complexity of handle multiple servers, by the other hand if we would like to scale out the application in individual server its a good a idea to maximize the performance of our server with clusters. anyway I think openshift will give us only one core by application